src/wtforms/validators.py

```diff
@@ -2,6 +2,7 @@
 
 import re
 import uuid
+from email_validator import validate_email, EmailNotValidError
```

Comment:
Should this be optional like `ipaddress`?

Reply:
Maybe, but I prefer just adding a hard dependency since email validation is very common. What are the benefits of making it optional?

Reply:
If you don't use email validation (I don't for many projects), you now have an extra dependency that you don't use. I'd prefer an optional dependency since email validation isn't required to making WTForms work.

I don't recall exactly why, but I think using CC0 for a license is potentially problematic, so forcing users to install email_validator could cause licensing issues as well.

Reply:
Made optional with a2a7afe5d3dc7712ab949a512ba13ead40eed779. Also, added extras for both `email_validator` and `ipaddress` dependencies so they can be installed at the same time as `wtforms`. Removed `locale` extra since `Babel` is already a required dependency, so there's no reason for having it in extras.